bombardier.go

```diff
@@ -441,6 +441,8 @@ func (b *bombardier) disableOutput() {
 }
 
 func main() {
+	proxyServer = os.Getenv("BOMBARDIER_PROXY")
```

Comment:
Should probably come from a command line argument

Reply:
done